ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -1,14 +1,16 @@
 <div class="replication-dashboard box is-sideless is-fullwidth is-marginless" data-test-replication-dashboard>
   {{#if isReindexing}}
-    <AlertBanner
-      @title={{concat "Re-indexing in progress" reindexingStage}}
-      @type="info"
-      @secondIconType="loading"
-      @message="This can cause a delay depending on the size of the data store. You can use Vault during this time."
-      data-test-isReindexing/>
+    <div class="reindexing-alert">
+      <AlertBanner
+        @title={{concat "Re-indexing in progress" reindexingStage}}
+        @type="info"
+        @progressBar={{progressBar}}
+        @message="This can cause a delay depending on the size of the data store. You can use Vault during this time."
+        data-test-isReindexing />
+    </div>
   {{/if}}
 
-  <div class="selectable-card-container has-top-margin-xl">
+  <div class="selectable-card-container">
```

Comment:
Curious where the margin is now coming from for this view: 

![Screen Recording 2020-05-12 at 11 12 AM](https://user-images.githubusercontent.com/6618863/81724484-ad547580-9441-11ea-81d9-357096972ee6.gif)

Reply:
excellent question! i removed the `has-top-margin-xl` classes from the AlertBanners and instead applied them to the dashboard component itself. this way all instances of the dashboard have consistent margins and paddings. that change is right here: https://github.com/hashicorp/vault/pull/8975/files#diff-ad14bf8f91b12ee633f19fcf48abf7caR24

before that change the primary & dr secondary dashboards had slightly different top margins, and i thought it was better to keep the style tightly coupled to the dashboard component itself vs. manually applying styles to other components.